controller.py

```diff
@@ -29,11 +29,11 @@ def get_current_maze():
     return current_maze
 
 
-def run_DFS_on_maze(run_X_times):
+def run_DFS_on_maze(run_X_times, solver1):
     """Used to run a resolver x amunt of times on the current maze"""
     global current_maze
     # model.Resolver(current_maze).solve(run_X_times)
-    resolve_maze(current_maze, amount=run_X_times, solver="dfs")
+    resolve_maze(current_maze, amount=run_X_times, solver=solver1)

```

Comment:
Forstår ikke helt den her funktion?
Kan du ikke bare kalde den direkte med dine parameter fra resolveren?

Reply:
Jeg forstår ikke hvad du mener, Ændringen her er jo at istedet for at have dfs med som standart så spørg jeg brugeren i view om han vil køre "dfs" eller "astar". Det er solver parameteren